services/prefect/flows/nyt_updater.py

```diff
@@ -0,0 +1,34 @@
+from prefect import Flow
+from prefect.schedules import CronSchedule
+
+from prefect.tasks.prefect import create_flow_run, wait_for_flow_run
+from prefect.utilities.notifications import slack_notifier
+from prefect.engine.state import Failed
+
+
+def init_updater_flow():
+    with Flow(
+        "NYTParquetUpdater",
+        schedule=CronSchedule("30 6 * * *"),
+        state_handlers=[slack_notifier(only_states=[Failed])]  # notify on failure only,
+    ) as flow:
+
+        # NYTimesCasesDeaths and UpdateParquetFiles flows must already be registered via
+        # generated_flows.py and update_api_view.py
```

Comment:
Not 100% sure what to make of this comment.  I assume if they're not, the code below will just fail?  I could imagine a more useful comment being something like:

> Note that the below code relies on NYTimesCasesDeaths and UpdateParquetFiles flows already being registered via generated_flows.py and update_api_view.py.  This is guaranteed by ...?

But I'm not actually sure what's guaranteeing it!  What controls the order of registration?  What are we doing to ensure they're registered in the right order?

Reply:
Yeah, I should make this more clear. 

Flows only need to be registered once per project, and once they've been registered they can be called/run whenever.  For most of the scraper flows registration happens in [`generated_flows.py`](https://github.com/covid-projections/can-scrapers/blob/main/services/prefect/flows/generated_flows.py#L151-L153), and for the update parquet flow it happens in [`update_api_view.py`](https://github.com/covid-projections/can-scrapers/blob/main/services/prefect/flows/update_api_view.py#L59-L66). So while it's not guaranteed, as long as those files have been run in the past (via `python update_api_view.py` or something) these flows should exist.